src/constants.js

```diff
@@ -0,0 +1,11 @@
+export const keys = {
```

Comment:
Please make sure that this file follows `editorconfig`.

Reply:
I am sorry, I don't seem to get what's wrong with the file, the indentation seems to be in sync with editorconfig, please suggest what is to be changed.

Reply:
Indent style should be space and not tabs.

Reply:
Done, please see

Reply:
Done, please check. Sorry for the trouble.